src/app/components/PlayersBoard.ts

```diff
@@ -0,0 +1,94 @@
+import { Render } from '../utils/Render';
+import { flatten } from 'lodash';
+import { Herd } from '.././logic/Herd';
+
+export class PlayersBoard {
+  /* returns player's board with player's name, avatar and herd */
+  renderPlayersBoard(
+    playersChosenAvatarPath: string,
+    playersChosenName: string,
+    playersColor: string,
+  ): void {
+    const playersBoardContainer: HTMLElement = Render.elementFactory(
+      'div',
+      {
+        className: 'board-container',
+      },
+    );
+    const playersDataPanel: HTMLElement = Render.elementFactory(
+      'div',
+      { className: 'players__data' },
+    );
+    const playerName: HTMLElement = Render.elementFactory(
+      'h3',
+      {
+        className: 'players__data__name',
+        style: `color:${playersColor}`,
+      },
+      playersChosenName,
+    );
+    const playerAvatar: HTMLElement = Render.elementFactory('img', {
+      className: 'players__data__avatar',
+      src: playersChosenAvatarPath,
+    });
+    const herd = new Herd();
+
+    const herdView: HTMLElement = Render.elementFactory('div', {
+      className: 'players__herd',
+    });
+
+    const playerHerd = herd.theAnimals.map(([animal, count]) => [
+      animal.theName,
+      count,
+    ]);
+    const herdImagesAndCounts = playerHerd.map(
+      ([nameElement, countElement]) => {
+        if (
+          nameElement === 'big dog' ||
+          nameElement === 'small dog'
+        ) {
```

Comment:
To be resolved: image of a small dog

Reply:
If we introduce a small dog avatar, this method will be half the length and more readable.